pom.xml

```diff
@@ -175,6 +186,12 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.dataformat</groupId>
+            <artifactId>jackson-dataformat-yaml</artifactId>
+            <version>2.4.3</version>
+            <scope>test</scope>
```

Comment:
I deliberately placed all yaml stuff in test scope to avoid run-time dependency on jackson-dataformat-yaml for these who don't need it. But of course I'd live it to your opinion if it may be beneficial to move the ```YamlObjectMapperFactory``` into the compile scope and thus provide the default yaml factory implementation for library users.